client/src/connection/studio/index.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { AxiosInstance } from "axios";
+
+function getStudioSessionRequest(connection: AxiosInstance) {
+  // Create a Date object representing the current date and time
+  const date = new Date();
+  // Get the time zone offset
+  const timeZoneOffset = date.getTimezoneOffset();
+  // Extract hours and minutes from the time zone offset
+  const hoursOffset = Math.floor(Math.abs(timeZoneOffset) / 60);
+  const minutesOffset = Math.abs(timeZoneOffset) % 60;
+  // Convert the time zone offset to the desired format (e.g., "GMT+02:00")
+  const formattedTimeZoneOffset = `GMT${
+    timeZoneOffset >= 0 ? "-" : "+"
+  }${hoursOffset.toString().padStart(2, "0")}:${minutesOffset
+    .toString()
+    .padStart(2, "0")}`;
+  // Create the request body
+  const studioSessionRequest = JSON.stringify({
```

Comment:
You _could_ return here instead of variable assignment

Reply:
done in f5253ce014a5fec429456474725f717e0f2609a0